ml4floods/data/worldfloods/lightning.py

```diff
@@ -73,7 +73,8 @@ def __init__(
         num_workers:int = 4,
         num_workers_val:int = 0,
         num_workers_test: int = 0,
-        filter_windows:Callable = None
+        filter_windows:Callable = None,
```

Comment:
Please add `filename_train_test` and `filter_windows` to the docstring of the class above